backport/action.yml

```diff
@@ -116,7 +117,7 @@ runs:
         git am --keep-cr --signoff << "D2LEOF"
         ${{ steps.source-pr-patch.outputs.result }}
         D2LEOF
-        git push origin $INPUT_DESTINATION_HEAD_BRANCH:$INPUT_DESTINATION_HEAD_BRANCH
+        git push https://${GITHUB_TOKEN}@github.com/${{ github.repository }} $INPUT_DESTINATION_HEAD_BRANCH:$INPUT_DESTINATION_HEAD_BRANCH

```

Comment:
If you don't do this it will use the default github token which just feels off since we are required to pass one into the action.